core/drivers/sub.mk

```diff
@@ -27,3 +27,4 @@ srcs-$(CFG_STM32_UART) += stm32_uart.c
 srcs-$(CFG_STM32_I2C) += stm32_i2c.c
 srcs-$(CFG_BCM_HWRNG) += bcm_hwrng.c
 srcs-$(CFG_BCM_SOTP) += bcm_sotp.c
+srcs-$(CFG_BCM_GPIO) += bcm_gpio.c
```

Comment:
Need to `$(call force,CFG_BCM_GPIO,y)` in `conf.mk` in #2927?

Reply:
In the other one I have comment "where is GPIO driver"  :) 
So first let one of these get merged.

Reply:
#2927 is merged. I re-based this patch and added 
`$(call force,CFG_BCM_GPIO,y) in conf.mk`
Thanks all for the reviews.